internal/util/projutil/project_util.go

```diff
@@ -226,6 +246,15 @@ func IsOperatorGo() bool {
 }
 
 func IsOperatorAnsible() bool {
+	// If the project has the new layout we will check the type in the config file
+	if kbutil.HasProjectFile() {
```

Comment:
We may want to move the two functions `HasProjectFile()` and `ReadConfig()` out of the `kubebuilder` package if they do not belong there.

Reply:
The both functions are defined in the `kbutil` and are used in other places. Sorry, but it is not clear for your suggestion here. Note, that has no reason for we remove them from `kbutil` and it is not part of the scope/goal of this PR.

Reply:
The relocation of the two functions can be in a separate follow up, that is for sure. However, if I'm local running an Ansible operator, I may get trapped by the `HasProjectFile()` function here:
https://github.com/operator-framework/operator-sdk/blob/master/internal/util/kubebuilder/project.go#L35

The error message is "Failed to read PROJECT file to detect kubebuilder project:" and I was running an Ansible operator. Isn't that confusing?

Reply:
It is because you probably added the PROJECT file manually when we should not. 
See that when we merge the PR #3295 we will have the Helm new layout built with plugins and also we will do the same for Ansible as well. 

- So, the run local when has not PROJECT file is the current/legacy layout where it works perfectly: 

 $ operator-sdk run local

INFO[0000] Running the operator locally; watching namespace "default" 
{"level":"info","ts":1593683488.696401,"logger":"cmd","msg":"Go Version: go1.13.12"}
{"level":"info","ts":1593683488.696457,"logger":"cmd","msg":"Go OS/Arch: darwin/amd64"}
{"level":"info","ts":1593683488.696461,"logger":"cmd","msg":"Version of operator-sdk: v0.18.0+git"}
{"level":"info","ts":1593683488.697037,"logger":"cmd","msg":"Watching single namespace.","Namespace":"default"}

- Then, if you clone the PR #3295 and also apply the changes made here you will be able to check it working as well with the new layout by running `make run`.

Reply:
Marking as solved.